estate/views/estate_menus.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <menuitem id="test_menu_root" name="Real Estate">
+        <menuitem id="test_first_level_menu" name="Advertisments">
```

Comment:
```suggestion
        <menuitem id="test_first_level_menu" name="Advertisments">
```

Dont hesitate to use a spellchecker, it is effort, but many people will thank you (not directly, but more like, they wont curse your code)

Reply:
you cna have it in the IDE directly